src/audio/music1.asm

```diff
@@ -144,7 +144,7 @@ Music1_Init:
 	ld [wMusicPanning], a
 	xor a
 	ld [wdd8c], a
-	ld [wde53], a
+	ld [wd0eb], a
```

Comment:
`wde53` is what it's still named in the tcg1 repo, so even though the address is different, I've kept the matching name so that it's clear that they in fact represent the same variable. That's a bit harder to remember/keep track of if their names are made different. This isn't a perfect system either though, so either way is fine.